src/components/App/App.js

```diff
@@ -76,61 +173,75 @@ function App(props) {
       }
   }, []);
 
+  useEffect(() => {
+    getSavedArticles();
+  }, [currentUser])
+
   //#endregion
 
 
   //#region Rendering
 
   return (
     <div className="page">
+    <CurrentUserContext.Provider value={currentUser}>
       <Header
```

Comment:
You need to format the markup to show **indentation** and to make the code more readable. You can do that with `prettier` or a combination `SHIFT + ALT + F`


```jsx

  return (
    <div className="page">
      <CurrentUserContext.Provider value={currentUser}>
        <Header

```